R/run_stress_test.R

```diff
@@ -331,9 +336,12 @@ st_read_agnostic <- function(dir, start_year) {
   return(out)
 }
 
-st_process <- function(data, asset_type, company_exclusion) {
+st_process <- function(data, asset_type, company_exclusion, scenario_data) {
   start_year <- get_start_year(data)
   scenarios_filter <- scenarios_filter()
+  declining_technologies <- scenario_data %>%
```

Comment:
edgecase question, what happens if there are not decllining technologies, does everthing run smooth?

Reply:
In the portfolio or in the scenario data? There will not be a scenario data set without declining technologies. Will check on the portfolio level though

Reply:
Same impact on company, if only increasing technologies are given

Reply:
I do not understand the last remark, so just to go sure. Let us assume we have portfolio that holds only 1 company in the renewables sector. The analysis will still run?

Reply:
The test was: portfolio with one company in the power sector. This company has assets in the Nuclear and Renewables technologies, both of which are `"increasing"`. Hence no, declining techs given. Still runs. But: the newest version should not contain the `declining_technologies` variable anymore anyway